proc_scripts/baseline.py

```diff
@@ -53,10 +56,10 @@ def costfun(ini_vec):
         d_test = apply_corr(ini_vec)
         return abs(d_test.real).sum('t2').data.item()
     max_val = abs(this_d_tdom.data).max()
-    print(max_val)
+    logger.info(strm(max_val))
     mybounds = r_[-max_val,max_val][newaxis,:]*ones(npts*2)[:,newaxis]
-    print(shape(mybounds))
-    print(mybounds)
+    logger.info(strm(shape(mybounds)))
+    logger.info(strm(mybounds))
```

Comment:
`logging.info` should work (see comment above)

Reply:
I would run `grep -rie 'init_logging' *` inside the `proc_scripts` subdirectory to find any files where this logging thing is an issue, and then fix it.